src/screens/components/Navbar.jsx

```diff
@@ -0,0 +1,36 @@
+import React from "react";
+import "../../App.css";
+
+export default function Navbar() {
+  return (
+    <div style={{ marginTop: "5px" }}>
+      <ul className="navbar-list">
```

Comment:
we probably should rely on flexbox instead of the line item html elements. 
```
<div flexDirection="row>
   <div className="navbar-list">
    //items with flex styling
   </div>
</div>
```

Reply:
this is non blocking, we can revisit later